main.go

```diff
@@ -229,34 +176,84 @@ func main() {
 	// Use the oauth client to authenticate to Github API
 	client := github.NewClient(tc)
 
-	passed := robot.Statistics.Total.Stat[0].Pass
-	failed := robot.Statistics.Total.Stat[0].Fail
-	total := len(robot.Suite.Suite.Test)
+	totalPassed := output.Statistics.Total.Stat[0].Pass
+	totalFailed := output.Statistics.Total.Stat[0].Fail
+	totalSkipped := output.Statistics.Total.Stat[0].Skip
+	total := len(output.Suite.Suite.Test)
+
+	passedInt, err := strconv.Atoi(totalPassed)
+	if err != nil {
+		log.Panic(err)
+	}
+
+	passPercentage := float64(passedInt) / float64(total) * 100
 
 	vars := make(map[string]interface{})
-	vars["Passed"] = &passed
-	vars["Failed"] = &failed
+	vars["Passed"] = &totalPassed
```

Comment:
These could be directly assigned without first setting them to variable as the values are not used elsewhere.